src/addons/dltx.md

```diff
@@ -30,18 +32,18 @@ If you want to delete a variable, simply add a line "!variable_name". You can de
 You can only override sections that already have a base definition. Both loading an override without a base section, or loading multiple base sections will cause crashes. Multiple overrides can be applied to the same section. If multiple mods override the same value, the one that is loaded last will win the conflict over that particular variable. I haven't verified what actually affects the load order, but I'm going to guess it's probably based on the alphabetic order of the file names.
 
 ### Additional DLTX functions (Modded Exes 1.5.2 only)
+
 [Modded Exes](https://github.com/themrdemonized/STALKER-Anomaly-modded-exes) enhances DLTX with providing additional functions:
 
 * DLTX received possibility to create section if it doesn't exists and override section if it does with the same symbol `@`.
   Below is the example for `newsection` that wasn't defined. Firstly its created with one param `override = false`, then its overriden with `override = true`
 
-  ```
+  ```ltx
```

Comment:
ltx is not an actual format (stalker community made it up), so it should be either ini or nothing at all (I prefer nothing at all)